recipe/meta.yaml

```diff
@@ -10,12 +10,15 @@ source:
   sha256: 6a495dbd5aac1f89d1ed51140fc41adb1f892293492b51b667eb5c40df1bde48
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py2k]
   script:
     - rm $PREFIX/include/cblas.h  # [not win]
     - del /f %LIBRARY_INC%\cblas.h  # [win]
-    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+    # build the C files in parallel. This helps beat the
+    # time limit consistently on drone.io
+    - {{ PYTHON }} setup.py build_ext -j{{ CPU_COUNT }}
+    - {{ PYTHON }} -m pip install . -vv
```

Comment:
Do we want to keep `--no-deps`?

Reply:
I'm 90 percent sure conda forge does this in their pip config

Reply:
A while back, I went on this hunt, and found that they isolate the environment:
https://github.com/scikit-image/scikit-image/issues/4261#issuecomment-557322707

Reply:
Actually [it's done in conda-build]( https://github.com/conda/conda-build/blob/1c76172835eec45ebaf5a0fa17dace84dc4e3475/conda_build/build.py#L2368 ), but yeah we don't need it here.

Reply:
https://github.com/conda/conda-build/pull/3271/files

Reply:
https://github.com/conda/conda-build/blob/master/CHANGELOG.txt#L577

Reply:
Thanks for the confirmation!